diff --git a/src/MediaTypeManager.php b/src/MediaTypeManager.php
index 5ec95cc..6652b04 100644
--- a/src/MediaTypeManager.php
+++ b/src/MediaTypeManager.php
@@ -26,7 +26,6 @@ class MediaTypeManager extends DefaultPluginManager {
     parent::__construct('Plugin/MediaEntity/Type', $namespaces, $module_handler, 'Drupal\media_entity\MediaTypeInterface', 'Drupal\media_entity\Annotation\MediaType');
 
     $this->alterInfo('media_entity_type_info');
-    $this->setCacheBackend($cache_backend, 'media_entity_type_plugins');
   }
 
 }
