diff --git a/lightning_api.install b/lightning_api.install
index f3023bc..40f1e26 100644
--- a/lightning_api.install
+++ b/lightning_api.install
@@ -9,6 +9,7 @@ use Drupal\Core\Field\BaseFieldDefinition;
 use Drupal\Core\Language\LanguageInterface;
 use Drupal\Core\StringTranslation\TranslatableMarkup;
 use Drupal\Core\Url;
+use Drupal\path_alias\Entity\PathAlias;
 use Drupal\views\Entity\View;
 
 /**
@@ -20,10 +21,10 @@ function lightning_api_install() {
   // If the openapi_ui_redoc and jsonapi modules are installed, give the API
   // documentation a nice path alias.
   if ($module_handler->moduleExists('openapi_ui_redoc') && $module_handler->moduleExists('jsonapi')) {
-    /** @var \Drupal\Core\Path\AliasStorageInterface $alias_storage */
-    $alias_storage = \Drupal::service('path.alias_storage');
+    /** @var \Drupal\path_alias\AliasRepository $alias_storage */
+    $alias_storage = \Drupal::service('path_alias.repository');
 
-    $path = $alias_storage->lookupPathSource(
+    $path = $alias_storage->lookupByAlias(
       '/api-docs',
       LanguageInterface::LANGCODE_NOT_SPECIFIED
     );
@@ -32,10 +33,10 @@ function lightning_api_install() {
         'openapi_ui' => 'redoc',
         'openapi_generator' => 'jsonapi',
       ];
-      $alias_storage->save(
-        Url::fromRoute('openapi.documentation', $route_parameters)->toString(),
-        '/api-docs'
-      );
+      PathAlias::create([
+        'path' => Url::fromRoute('openapi.documentation', $route_parameters)->toString(),
+        'alias' => '/api-docs',
+      ])->save();
     }
   }
 
